src/components/tabController/TabBarItem.tsx

```diff
@@ -226,7 +230,20 @@ export default class TabBarItem extends PureComponent<Props> {
   }
 
   getIconStyle() {
-    const {index, currentPage, iconColor, selectedIconColor, labelColor, selectedLabelColor, ignore} = this.props;
+    const {
+      index,
+      currentPage,
+      iconColor,
+      selectedIconColor,
+      labelColor,
+      selectedLabelColor,
+      ignore,
+      disableIconTintColor
+    } = this.props;
+
+    if (disableIconTintColor) {
+      return {};
```

Comment:
suggest return undefined here and not an empty object. 
new object with a new ref might trigger a render, while undefined won't